pdb2pqr/io.py

```diff
@@ -6,7 +6,7 @@
 from collections import Counter
 from pathlib import Path
 from sys import path as sys_path
-import requests
+import gemmi 
```

Comment:
`gemmi` is a new dependency. I think it's a great library, but adding dependencies should be carefully thought through. Is https://github.com/Electrostatics/mmcif_pdbx `pdbx.dump` not suitable to write out the modified file?

Reply:
I briefly looked into PDBx but could not find any good documentation on how to append new columns to the atom_site table. However, I did not see the Electrostatics PDBx repo and found other ones on the internet. 

I can check this repo out this weekend and see if it can easily be transitioned from gemmi to PDBx.

Reply:
Thank you -- https://github.com/Electrostatics/mmcif_pdbx and the pdb2cif library I'm working on are the preferred mechanisms for this interconversion.

Reply:
So I refactored from gemmi to pdbx. 

But the PR is now closed??

Reply:
My apologies -- I didn't realize you were planning to do that in the same PR since it seemed like a major change.  I'll reopen.

Reply:
That being said, my long-term goal is to use https://github.com/Electrostatics/pdb2cif for the conversion process.